articles/architecture-scenarios/b2c/planning.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Planning Your Consumer Identity and Access Management Integration
+description: Why it's important to plan your Consumer Identity and Access Management Implementation prior to beginning the integration
+toc: false
+topics:
+    - b2c
+    - ciam
+contentType: concept
+useCase:
+  - implementation
+---
+# Planning Your Consumer Identity and Access Management Integration
+
+In our experience, customers who establish clear objectives and align those objectives to requirements early on benefit the most from their integration with Auth0. Regardless of the project type, we recommend that you plan your project using a phased approach across multiple workstreams. In most cases, we see the most success with those adopting a three-phase project structure.
+
+::: note
+Identifying your primary objectives early on during the project will help you focus on the specifics important to building out your solution. For instance, if your primary objective is to avoid disruption to end users, then you should not adopt a "big bang" approach when integrating with Auth0.
+:::
+
+We also think that customers who adopt an iterative release process make better progress. For example, let's say that you have 3-4 applications you want to integrate with Auth0. Rather than integrating all at once and shipping only when that's done, you might consider completing the integration for one application before moving on to the next. This way, your teams can benefit from the experience of working with Auth0 and leverage this experience to help increase the velocity of each iteration.
```

Comment:
```suggestion
The evidence we've collected suggests that customers who also adopt an iterative release process make better progress. For example, let's say that you have 3-4 applications you want to integrate with Auth0. Rather than integrating all at once and shipping only when that's done, you might consider completing the integration for one application before moving on to the next. This way, your teams can benefit from the experience of working with Auth0 and leverage this experience to help increase the velocity of each iteration.
```